.github/CODEOWNERS

```diff
@@ -1,41 +1,2 @@
-/src/lib/protocols/ @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @yufengwangca @pan-apple
-/src/lib/shell/  @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @mspang @turon
-/src/lib/support/  @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @rwalker-apple @gerickson @pan-apple
-/src/platform/  @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @pan-apple  @mspang @vivien-apple @yufengwangca
-/src/qrcodetool/  @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @shana-apple
-/src/setup_payload/  @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @bhaskar-apple @vivien-apple
-/src/system/  @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @sagar-apple @pan-apple @gerickson @turon @rwalker-apple
-/src/test_driver/  @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @pan-apple @vivien-apple
-/src/transport/  @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @andy31415 @pan-apple @vivien-apple @mspang
-
-# infra directories
-/.github/ @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @woody-apple @andy31415
-/.devcontainer/ @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @woody-apple @rwalker-apple
-/integrations/docker/ @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @rwalker-apple @woody-apple @andy31415
-/scripts/ @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @rwalker-apple @woody-apple @andy31415 @mspang
-
-# this catches any directory named docs and its subdirs
-docs/ @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @gerickson @woody-apple @andy31415 @rwalker-apple
-*.md @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @gerickson @woody-apple @andy31415 @rwalker-apple
-
-# GN make system
-/.gn/ @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @mspang
-*.gn @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @mspang
-*.gni @chrisdecenzo @woody-apple @bzbarsky-apple @hawk248 @jelderton @andy31415 @mspang @saurabhst @BroderickCarlin @mspang
+* @reviewers-amazon @reviewers-apple @reviewers-comcast @reviewers-google @reviewers-samsung
```

Comment:
hope this line works, we had trouble with it while you were on vacation, but perhaps that was because there were more lines in this file confusing github

Reply:
worked before :)